src/main/java/course_project/Main.java

```diff
@@ -0,0 +1,11 @@
+package course_project;
+
+import course_project.service.Runner;
+
+public class Main {
+    public static void main(String[] args) {
+        Runner runner = new Runner();
```

Comment:
Seems broken:
![image](https://user-images.githubusercontent.com/58872509/134480519-a87564a0-7ae7-4b7e-84ab-93c55ee7ffc5.png)
Also, please introduce automatic ship placement

Reply:
Also, if input is wrong, the app just shuts down. Really annoying in case you make a typo right before the end :)

Reply:
![image](https://user-images.githubusercontent.com/58872509/134553330-e4d57862-088b-41fb-a6ee-b4c6f74afe27.png)

Reply:
![image](https://user-images.githubusercontent.com/58872509/134553361-ca06edc8-a2bc-4f1f-a916-13c70c805e27.png)

Reply:
Have you played your own game?) 
Sorry, can't approve a this state